src/components/ScriptExplorer/OutputEntry.jsx

```diff
@@ -246,6 +255,21 @@ class OutputEntry extends React.Component {
             error={this.hasAmountError()}
             helperText={amountError}
             InputProps={{
+              startAdornment: (
+                <InputAdornment position="start">
+                  {isWallet &&
+                    autoSpend &&
+                    number === outputs.length &&
+                    number !== changeOutputIndex && (
```

Comment:
Should we also check that the spendable balance is > 0?

Reply:
There’s nothing that stores spendable balance at the moment, just spendable slices, but I could add a new selector.

Also probably would want to check that it's greater than Dust. 

A reason I could think about not putting this is that it's kind of superfluous since the tx wouldn't validate anyway and would still require extra calculations. Also, I probably wouldn't want to hide it if spendable balance isn't large enough, but just disable it. These conditionals are more about checking if this is the right output field to show it on (is it that last one, is it not change, are we in auto spend) rather than if it can be used to add a value.

Reply:
We can revisit this in a later PR if the functionality is desired.  It's not big enough of a deal to hold this up.